### DISCUSSION TITLE
Feature Request: Option to choose "Start Browserosaurus on Startup"

### DISCUSSION THREAD

**Discussion by dhawan1997** [will-stone/browserosaurus#493] [Ideas]
Hi,

I wanted to request if you could add an option to open the app on startup as that would save time. 

Sometimes I startup the computer and then forget to start this app separately as most apps offer that functionality. Only to realize when i click on a link and theres some delay. Because first the app need to open . It would make the experience more convenient.

**Comment by will-stone** [comment]
Hi, thanks for using Browserosaurus. Please see https://github.com/will-stone/browserosaurus/discussions/456
